.github/workflows/athena_test.yml

```
@@ -41,3 +41,24 @@ jobs:
     - name: Run tests
       working-directory: athena
       run: poetry run test_all
+
+    - name: Upload JUnit Test Results
+      if: success() || failure()
+      uses: actions/upload-artifact@v4
+      with:
+        name: Athena JUnit Test Results
+        path: athena/test-results/*.xml  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)